src/settings-ui/Settings.UI.UnitTests/BackwardsCompatibility/TestFiles/V0.21.1/Microsoft/PowerToys/Image Resizer/settings.json

```diff
@@ -39,7 +39,7 @@
         {
           "Id": 1,
           "ExtraBoxOpacity": 100,
-          "EnableEtraBoxes": true,
+          "EnableExtraBoxes": true,
```

Comment:
No idea if this would break existing setting files? I defer to @jaimecbernardo for that :)

Reply:
@stefansjfw FYI

Reply:
Let's revert this and merge the PR. I'll revisit it separately.  Thanks for pointing out

Reply:
Alright then, I'll let you pick this up.